### PR TITLE
Request change: lngo request change scorecard information

### PR DESCRIPTION
**Pull request by hengsokly**

### CODE REVIEW

File: app/controllers/scorecards/request_changes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      if @request_change.save
+        flash[:notice] = t("scorecard.create_successfully")
+        redirect_to scorecard_url(@scorecard.uuid)
+      else
+        render :new
+      end
+    end
+
+    def update
+      if @request_change.update(request_change_params)
```
Comment (kakada) [inline app/controllers/scorecards/request_changes_controller.rb:27]: @hengsokly please check the authorization for this, :update, and :approve actions

Comment (hengsokly) [reply app/controllers/scorecards/request_changes_controller.rb:27]: I already check it in find_request_change

def find_request_change
    @request_change = authorize @scorecard.request_changes.find(params[:id])
end

File: app/policies/scorecard_policy.rb [hunk 2]
```diff
@@ -40,6 +40,10 @@ def setting?
     user.program_admin? || user.staff? || user.lngo?
   end
 
+  def request_change?
+    user.lngo? || (create? && record.request_changes.length > 0)
```
Comment (kakada) [inline app/policies/scorecard_policy.rb:44]: @hengsokly request change should allow only the LNGO as staff and program admin is allowed by default without the need to go through the request change process, does it make sense?

File: app/views/languages/index.haml [hunk 3]
```diff
@@ -27,8 +27,7 @@
                 %td= language.name_km
                 %td= timeago(language.created_at).html_safe
                 %td.text-right
-                  .d-flex.d-md-block.flex-row
-                    = render 'shared/action_buttons', edit_url: edit_language_path(language), delete_url: language_path(language), object: language, remote: true
+                  = render 'shared/action_buttons', edit_url: edit_language_path(language), delete_url: language_path(language), object: language, remote: true
```
Comment (kakada) [inline app/views/languages/index.haml:30]: @hengsokly flex-row seems it used for responsive screen, please double check if removes is fine with responsive screen

Comment (hengsokly) [reply app/views/languages/index.haml:30]: No, this class is only for saying flex-direction, but d-flex class by default it is flex-row direction

File: app/policies/scorecard_policy.rb (hunk 2 above)
Comment (kakada) [reply app/policies/scorecard_policy.rb:44]: this is applied for the scorecard request change action button UI

**Review by kakada — APPROVED** [review]

**Pull request merged**